server/controllers/Events/resolver.ts

```diff
@@ -80,4 +82,12 @@ export class EventResolver {
 
     return true;
   }
+
+  validateEventUrl(url) {
+    try {
+      new URL(url);
```

Comment:
using `new URL()` is actually quite restrictive, and I think we want to be a little more permissive.  For example, the following will throw errors, despite being "fine" URLs:

```
new URL('google.com')
new URL('www.google.com')
```

Furthermore, valid URLs via `new URL()` include protocols we don't want to support, like `ftp` and `mailto`.

Suggestion: Before doing the try-catch, prepend it with `http://` if it isn't already, and if it passes, ensure the result's protocol is `http:` or `https:`  Let me know if this makes sense.

Reply:
I don't know how much value this is going to add, if we prepend http to the input, `new URL` will accept just about anything. We could try regex-based validation, but do we really need to be restricting what users enter in here? I think we're more likely to end up preventing someone from entering something they want to have in there than helping them with this validation

Reply:
Fair - I do think this is a nice to have and not at all a requirement, especially since we don't really validate any other fields.  I'd be fine with not adding any validation for now, and adding it later when we have better testing so we can also test it.

Reply:
If we're skipping the validation, is it safe to assume the framework handles sanitizing output so we're not subject to XSS on this field or any other user input for that matter?